setup_pulp.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+#
+# This script sets up a pulp all-in-one container (per the instructions at
+# https://pulpproject.org/pulp-in-one-container/) that the builder tests can
+# then use as an image repository that supports image signatures.
+#
+
+set -ex
```

Comment:
Would recommend setting `-u` to catch unset variables.

Cherry on top: `set -eu -o pipefail`

But since this script has neither vars nor I would find it understandable if we keep it like that

Reply:
Added `-u`